src/main.rs

```diff
@@ -432,6 +443,9 @@ fn compile_executable(target: &str, host: &host::Host) -> Result<(), io::Error>
     stdlib
     trace
     transform {
+      coalesce_ctrs
+      encode_adts
+      eta_reduce
```

Comment:
these are duplicated, also out of alphabetical order

Reply:
most likely a problem after the rebase